cmake/common.cmake

```diff
@@ -373,7 +373,12 @@ else()
   set(CONSENSUS_ARG "raft")
 endif()
 
-set(WORKER_THREADS 0)
+if("Release" IN_LIST CMAKE_BUILD_TYPE AND NOT PBFT)
```

Comment:
This won't work I believe, our release builds are actually RelWithDbgInfo or something like that.

Reply:
`RelWithDebInfo`

Reply:
`CMAKE_BUILD_TYPE` should be a single var, not a list, and I think this should actually be a `NOT Debug` check (ie - we want 0 worker threads in `Debug`, but 2 for everything else).

https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html